addons/l10n_ch/data/l10n_ch_chart_data.xml

```diff
@@ -5,7 +5,7 @@
     <data noupdate="1">
          <!-- Account Tags -->
         <record id="l10nch_chart_template" model="account.chart.template">
-            <field name="name">Plan comptable 2015</field>
+            <field name="name">Plan comptable 2015 (Suisse)</field>
```

Comment:
gné?

Reply:
When you have to choose your chart of account, you only have a list with multiple names. Let's say you have "Plan Comptable" in Belgium, "Plan Comptable 2019" in France and "Plan Comptable 2015" in Switzerland.
How do you choose the right one when having only the name of the CoA?

Reply:
I changed it here because I had the problem while testing with multiple CoA, and I was lucky the other CoA added the name of the country in it.